src/main/java/frc/team4909/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -1,19 +1,27 @@
 package frc.team4909.robot.subsystems.intake;
 
+import edu.wpi.first.wpilibj.AnalogInput;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 
 import edu.wpi.first.wpilibj.command.InstantCommand;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 public class IntakeSubsystem extends Subsystem{
+    double irSensorThreshold = 1.7;
```

Comment:
Please add a comment about what this constant means and how the threshold value was derived.